ogc_ws/src/despatcher/scripts/regular.py

```diff
@@ -115,14 +147,18 @@ def __init__(self):
             "airspeed": 0,
             "alt": 0,
             "arm": 0,
+            "mode": 0, 
             "groundspeed": 0,
+            "fuel": 0,
+            "batt": 0, 
```

Comment:
Will like to change this to alphabetical order (regular payload is sent out in alphabetical order of the entries). Makes it easier for us to refer to which entry number is which

Reply:
Noted on that sure will re-arrange the regular payload